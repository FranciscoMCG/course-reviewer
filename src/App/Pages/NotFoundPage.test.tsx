import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";

import NotFoundPage from "./NotFoundPage";
import { isRegExp } from "util";

describe("<NotFoundPage/ >", () => {
  it("should render component", () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
