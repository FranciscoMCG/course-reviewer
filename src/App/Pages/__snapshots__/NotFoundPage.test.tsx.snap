// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage/ > should render component 1`] = `
<p>
  Not Found
</p>
`;
